app/controllers/likes_controller.rb

```diff
@@ -4,7 +4,8 @@ def create
     @user_id = params[:user_id]
     @post_id = params[:post_id]
     if liked?
-      redirect_to user_post_path(@user_id, @post_id)
+      # redirect_to user_post_path(@user_id, @post_id)
```

Comment:
- [ ] As requested by the previous reviewer kindly remove the commented code to make your code cleaner